telegram/games/game.py

```diff
@@ -120,25 +121,25 @@ def parse_text_entity(self, entity):
 
     def parse_text_entities(self, types=None):
         """
-        Returns a ``dict`` that maps :class:`telegram.MessageEntity` to ``str``.
+        Returns a :obj:`dict` that maps :class:`telegram.MessageEntity` to :obj:`str`.
         It contains entities from this message filtered by their ``type`` attribute as the key, and
```

Comment:
Alright I guess.. length of docstrings really doesn't matter like at all, but here it's fine since the link will be right above anyways.